extensions/wikia/PhalanxII/fallback/hooks/TitleBlock.class.php

```diff
@@ -90,28 +90,28 @@ static public function genericTitleCheck( $titleObj, &$block ) {
 	static public function checkTitle( Title $title, &$block ) {
 		wfProfileIn( __METHOD__ );
 
-		if (is_null(self::$blocksData)) {
+		if ( is_null( self::$blocksData ) ) {
 			self::$blocksData = PhalanxFallback::getFromFilter( PhalanxFallback::TYPE_TITLE );
 		}
 		$fullText = $title->getFullText();
-		$result = array('blocked' => false);
+		$result = array( 'blocked' => false );
 
-		if ( !empty(self::$blocksData) ) {
+		if ( !empty( self::$blocksData ) ) {
 			$blockData = null;
 			$result = PhalanxFallback::findBlocked( $fullText, self::$blocksData, true, $blockData );
 			if ( $result['blocked'] ) {
 				$block = ( object ) $blockData;
-				#self::spamPage( $result['msg'], $title );
-				Wikia::log(__METHOD__, __LINE__, "Block '{$result['msg']}' blocked '$fullText'.");
+				# self::spamPage( $result['msg'], $title );
+				Wikia::log( __METHOD__, __LINE__, "Block '{$result['msg']}' blocked '$fullText'." );
 			}
 		}
 
 		wfProfileOut( __METHOD__ );
 		return !$result['blocked'];
 	}
 
-	//moved from SpamRegexBatch.php
-	//TODO: use Phalanx messages
+	// moved from SpamRegexBatch.php
+	// TODO: use Phalanx messages
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)